tools/firtool/firtool.cpp

```diff
@@ -323,6 +323,8 @@ static LogicalResult processBuffer(
   if (failed(applyPassManagerCLOptions(pm)))
     return failure();
 
+  pm.nest<firrtl::CircuitOp>().addPass(firrtl::createResolvePathsPass());
+
   // Legalize away "open" aggregates to hw-only versions.
   pm.nest<firrtl::CircuitOp>().addPass(firrtl::createLowerOpenAggsPass());
 
```

Comment:
LowerOpenAggs does not support annotations, so:
```suggestion
  // Legalize away "open" aggregates to hw-only versions.
  pm.nest<firrtl::CircuitOp>().addPass(firrtl::createLowerOpenAggsPass());

  pm.nest<firrtl::CircuitOp>().addPass(firrtl::createResolvePathsPass());
```